### PR TITLE
com.discordapp.DiscordPTB

### PR DESCRIPTION
**Pull request by kuflierl**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*    **Access to camera is highly recommended for discord. Filesystem ro access to Videos and Pictures is required for selecting files. Filesystem rw access is required for downloading files**
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* **The updater is not compatible with flatpak**

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by kuflierl** [comment]
bot, build com.discordapp.DiscordPTB

**Comment by hfiguiere** [comment]
PR is supposed to be done against the `new-pr` branch

**Comment by hfiguiere** [comment]
what's the difference with https://flathub.org/apps/details/com.discordapp.Discord ?

**Comment by kuflierl** [comment]
> PR is supposed to be done against the `new-pr` branch

Oh, my bad. Should i close and do the PR again?

File: com.discordapp.DiscordPTB.json [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+			"name": "discord-ptb",
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -Dm755 discord-ptb.sh /app/bin/discord-ptb",
+				"install -Dm755 set-gtk-dark-theme.py /app/bin",
+				"install -Dm755 disable-breaking-updates.py /app/bin",
+				"mv DiscordPTB /app/discord-ptb",
+				"chmod +x /app/discord-ptb/DiscordPTB",
+				"install -d /app/share/applications",
+				"sed -e 's/Icon=discord-ptb/Icon=com.discordapp.DiscordPTB/' -e 's|Exec=/usr/share/discord-ptb/DiscordPTB|Exec=discord-ptb|' /app/discord-ptb/discord-ptb.desktop > /app/share/applications/${FLATPAK_ID}.desktop",
```
Comment (hfiguiere) [inline com.discordapp.DiscordPTB.json:99]: you should use `desktop-file-edit` instead.

**Comment by kuflierl** [comment]
> what's the difference with https://flathub.org/apps/details/com.discordapp.Discord ?

It is the Public beta of Discord.

**Comment by hfiguiere** [comment]
so instead it should be on the 'beta' branch of the other one.

also is redistribution allowed?

**Comment by kuflierl** [comment]
> so instead it should be on the 'beta' branch of the other one.

I don't think it should be since discord has given the programs separate names and desktop files. Also the current maintainer does not want to support it: https://github.com/flathub/com.discordapp.Discord/issues/226

**Comment by kuflierl** [comment]
> also is redistribution allowed?

I assume it is ok since com.discordapp.Discord exists.

File: com.discordapp.DiscordPTB.json (hunk 1 above)
Comment (kuflierl) [reply com.discordapp.DiscordPTB.json]: done

**Comment by kuflierl** [comment]
bot, build com.discordapp.DiscordPTB

**Comment by kuflierl** [comment]
i am going to do the pr against new-pr now. Closing

**Pull request closed without merging**